views/audit/query_audit_v2.sql

```diff
@@ -0,0 +1,334 @@
+/* Public BigQuery Audit View */
```

Comment:
Sometimes the BigQuery built-in SQL formatter is not great. This is one of those times. Please format this to a bit more readable